serial-studio: add 1.0.8

umarcor opened the pull request:
This PR adds [Serial-Studio/Serial-Studio](https://github.com/Serial-Studio/Serial-Studio): a "*multi-purpose serial data visualization & processing program*".

![Serial-Studio](https://raw.githubusercontent.com/Serial-Studio/Serial-Studio/master/doc/app-usage.gif)

Biswa96 commented:
Some comments:

1. Use lower case for the `_realname` variable.
2. If possible, a separate build folder is preferred.
3. In `package()` step, instead of manually creating paths with `mkdir` and `cp` commands, `install -D` command with other options is better choice, IMO.

umarcor commented:
@Biswa96, I addressed your concerns. At the same time, a new version was tagged some minutes ago, so I updated it.

The build error is arbitray. The build needs to be restarted.

umarcor commented:
See successful executions in my fork (same commit):

- https://github.com/umarcor/MINGW-packages/actions/runs/492257786
- https://github.com/umarcor/MINGW-packages/actions/runs/492224111
